self-core-impl/src/main/java/com/selfxdsd/core/GithubIssueComments.java

```diff
@@ -57,7 +57,7 @@
         if(issueUriStr.endsWith("/")){
             slash = "";
         }
-        this.commentsUri = URI.create(issueUriStr + slash + "comments");
+        this.commentsUri = URI.create(issueUriStr + slash + "comments/");
```

Comment:
@amihaiemil I've added `/` back, on my end some tests failed without it

Reply:
@criske trailing ``/`` is not ok, Github returns 404 because of it -- I found this bug and tested manually, but I was too lazy to write a test, sorry :D 

So please remove it and add a todo to fix the tests later.